tests/circuit_graph/test_circuit_graph.py

```diff
@@ -92,6 +94,27 @@ def circuit_measure_multiple_with_max_twice():
     )
 
 
+class CustomOpDepth2(ResourcesOperation):
+    num_wires = 3
+
+    def resources(self):
+        return Resources(num_wires=self.num_wires, depth=3)
+
+
+class CustomOpDepth3(ResourcesOperation):
+    num_wires = 2
+
+    def resources(self):
+        return Resources(num_wires=self.num_wires, depth=3)
+
+
+class CustomOpDepth4(ResourcesOperation):
+    num_wires = 2
+
+    def resources(self):
+        return Resources(num_wires=self.num_wires, depth=3)
```

Comment:
Did we mean to have 
```suggestion
        return Resources(num_wires=self.num_wires, depth=2)


class CustomOpDepth3(ResourcesOperation):
    num_wires = 2

    def resources(self):
        return Resources(num_wires=self.num_wires, depth=3)


class CustomOpDepth4(ResourcesOperation):
    num_wires = 2

    def resources(self):
        return Resources(num_wires=self.num_wires, depth=4)
```

Did we mean to have different depths here?

Reply:
Yes! Nice catch!